index.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
 
     return ESLint.create(tree, {
       testGenerator: this.options.testGenerator || this._testGenerator,
-      group: (this.options.group !== false) ? type : undefined,
+      group: (this.options.testGenerator && this.options.group !== false) ? type : undefined,
```

Comment:
this does not work if the test generator is detected automatically by `this._testGenerator`.

I guess the logic should be:

use `group: type` if:

- `group: true` was explicitly specified
- a `testGenerator` was specified or automatically detected that supports `group` (will have to build something like `ESLint.supportsGrouping(testGeneratorName)` into https://github.com/ember-cli/broccoli-lint-eslint)

and otherwise set the `group` option to `undefined`